cypress/e2e/swap/errors.test.ts

```diff
@@ -90,18 +99,25 @@ describe('Swap errors', () => {
         cy.contains('Confirm swap').click()
         cy.wait('@eth_sendRawTransaction').wait('@eth_getTransactionReceipt')
         cy.contains('Swap submitted')
-        cy.get(getTestSelector('confirmation-close-icon')).click()
+        if (i === 0) {
+          cy.get(getTestSelector('confirmation-close-icon')).click()
+        }
```

Comment:
Is this branching necessary?

Reply:
yes we only want to dismiss the first modal. i could take the code out of a loop to avoid the branching